content/en/docs/tasks/debug/debug-cluster/resource-metrics-pipeline.md

```diff
@@ -50,7 +50,7 @@ P[pod data] -.- C
 end
 L[API<br>server]
 W[HPA]
-C ---->|Summary<br>API| A -->|metrics<br>API| L --> W
+C ---->|metrics<br>resource| A -->|metrics<br>API| L --> W
```

Comment:
```suggestion
C ---->|metrics resource<br>API| A -->|metrics<br>API| L --> W
```
It's easier to understand.


Have any suggestions？@sftim  @tengqm

Reply:
Agree with the suggested change. 
- `Metrics resource api` is clearer than `metrics resource` for `/metrics/resource` API.

Reply:
```suggestion
C ---->|node level<br>resource metrics| A -->|metrics<br>API| L --> W
```